yb-voyager/cmd/endMigrationCommand.go

```diff
@@ -210,7 +210,7 @@ func saveMigrationReportsFn(msr *metadb.MigrationStatusRecord) {
 func saveMigrationAssessmentReport() {
 	assessmentReportGlobPath := filepath.Join(backupDir, "reports", fmt.Sprintf("%s.*", ASSESSMENT_FILE_NAME))
 	alreadyBackedUp := utils.FileOrFolderExistsWithGlobPattern(assessmentReportGlobPath)
-	migrationAssessmentDone, err := IsMigrationAssessmentDone(metaDB)
+	migrationAssessmentDone, err := IsMigrationAssessmentDoneDirectly(metaDB)
```

Comment:
Why not save the report in the case of assessment done if invoked by export-schema?

Reply:
didn't get you